drivers/modem/ublox-sara-r4.c

```diff
@@ -1766,4 +1766,3 @@ NET_DEVICE_OFFLOAD_INIT(modem_sara, CONFIG_MODEM_UBLOX_SARA_R4_NAME,
 			modem_init, &mdata, NULL,
 			CONFIG_MODEM_UBLOX_SARA_R4_INIT_PRIORITY, &api_funcs,
 			MDM_MAX_DATA_LENGTH);
-
```

Comment:
Just squash the "drivers: modem: ublox-sara-r4: removed blank line" commit to previous one.

Reply:
@jukkar Sorry - tried to do that, but it only added more commits. Should I clear this up some way or another?

Reply:
Just squash the commits together, and force push to this branch. Do not close this PR and resubmit, it is just pointless.

Reply:
Are you trying to do this from github web ui? That (squashing, force pushing) is probably not going to work from web ui, one needs to use git command line for that.

Reply:
got it - thanks for the comments.